superset-frontend/src/components/AlteredSliceTag/index.jsx

```diff
@@ -183,9 +193,7 @@ export default class AlteredSliceTag extends React.Component {
   renderTriggerNode() {
     return (
       <Tooltip id="difference-tooltip" title={t('Click to see difference')}>
-        <span className="label label-warning" style={{ fontSize: '12px' }}>
-          {t('Altered')}
-        </span>
+        <StyledLabel className="label">{t('Altered')}</StyledLabel>
```

Comment:
Unfortunately it's still needed as I wasn't able to migrate to the `Label` component yet due to the inconsistent colors (the outline that is rendered if the Label has a click handler is way too dark right now; we chatted about it earlier today with Kasia):

![image](https://user-images.githubusercontent.com/33317356/165525529-3802e982-9be2-4737-b972-92c83d129e9e.png)

But once that gets sorted This will be completely refactored to use the `Label` component.